Gruntfile.js

```diff
@@ -0,0 +1,35 @@
+module.exports = function(grunt) {
+
+  // Project configuration.
+  grunt.initConfig({
+    pkg: grunt.file.readJSON('package.json'),
+    uncss: {
+      dist: {
+        options: {
+          stylesheets: [
+            'assets/css/styles.css'
+          ]
+        }
+        files: {
+          'tidy.css': [
```

Comment:
Expected ')' and instead saw 'tidy.css'.<br>Missing semicolon.<br>Expected '}' to match '{' from line 1 and instead saw ':'.